### DISCUSSION TITLE
Add option for changing inline code color

### DISCUSSION THREAD

**Discussion by zenggyu** [quarto-dev/quarto-cli#6780] [Q&A]
### Description

Currently, quarto html format styles inline code with a predefined color, in addition to a background color. When there are just a few inline code in a document, this is very useful for distinguishing inline code from ordinary text. However, when a document contains many inline code, the styling tend to be distracting. E.g., here is a screen shot from one of my [blog posts](https://blog.zenggyu.com/posts/en/2018-01-29-installing-r-r-packages-e-g-tidyverse-and-rstudio-on-ubuntu-linux) (this may not seem terrible, but imagine if there are more inline code):

![图片](https://github.com/quarto-dev/quarto-cli/assets/10095784/b2b3836d-ef1c-4e24-883d-feb9f5502c63)

I am aware that there is a `monobackgroundcolor` option for html output, and I can set it to "transparent" to partly resolve the problem, but I think there should also be an option (e.g., `monoclolor`) to remove the color of the inline code (or perhaps change to other color).

**Comment by mcanouil** [comment]
You mean the following?
- https://quarto.org/docs/output-formats/html-themes.html#inline-code
ALso note that colouring is based on the highlight style you've chosen, see <https://quarto.org/docs/output-formats/html-code.html#highlighting>.

PS: I am changing this as a Q&A since there are already options for HTML.

**Reply by zenggyu** [reply]
It seems that the solution you provided can only be used with an additional SCSS file. I wonder if there will be a YAML option for changing mono font color, just like `monobackgroundcolor` for changing mono font background color?

**Reply by mcanouil** [reply]
I made a PR to start a discussion on this: https://github.com/quarto-dev/quarto-cli/pull/6969

**Comment by mskyttner** [comment]
I found this after lengthy searching. So nice with this PR. 

Would it be possible to in addition to being able to adjust this through yaml settings also provide default settings with more contrast (bg vs fg) for the inline code color, and perhaps is less colorful (and more boring) using grayscales? 

To illustrate with a comparison to what I get now, without making any particular settings:

![image](https://github.com/quarto-dev/quarto-cli/assets/1715840/9189730d-3c55-49c6-a00d-daddcd48c367)

This way provides more contrast and doesn't jump out as much:

![image](https://github.com/quarto-dev/quarto-cli/assets/1715840/30123150-1487-4de2-a5be-06cad6f0194c)

**Reply by mcanouil** [reply]
See the PR description.

Contrast is up to you: your HTML theme, your highlight style colour scheme, etc.
